doc/release-notes.md

```diff
@@ -4,3 +4,31 @@ release-notes at release time)
 Notable changes
 ===============
 
+RPC Changes
+-----------
+
+- `getdeprecationinfo` has several changes:
+  - It now returns additional information about currently deprecated and
+    disabled features.
+  - A new `end_of_service` object that contains both the block height for
+    end-of-service and the estimated time that the end-of-service halt is
+    expected to occur. Note that this height is just an approximation and
+    should not be relied upon due to the variability in block times. The
```

Comment:
If it shouldn't be relied upon, why add it? I think the more relevant point is that users shouldn't assume that a point-in-time reading of the value will remain correct:
```suggestion
    expected to occur. Note that this height is just an approximation and
    will change over time as the end-of-service block height approaches,
    due to the variability in block times. The
```